Dockerfile

```diff
@@ -23,25 +23,25 @@ RUN apt-get update && \
 COPY package.json package-lock.json ./
 RUN npm install
 
-ADD Gemfile* ./
+COPY Gemfile* ./
```

Comment:
changed this to COPY both for security reasons and to improve caching (was slowing down my development when I had to do some container rebuilds). should have no noticeable impact besides container build speed